// cielo/dist/enums.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var EnumIdentityType;
(function (EnumIdentityType) {
    EnumIdentityType["RG"] = "RG";
    EnumIdentityType["CPF"] = "CPF";
    EnumIdentityType["CNPJ"] = "CNPJ";
})(EnumIdentityType = exports.EnumIdentityType || (exports.EnumIdentityType = {}));
var EnumPaymentInterest;
(function (EnumPaymentInterest) {
    EnumPaymentInterest["MERCHANT"] = "ByMerchant";
    EnumPaymentInterest["ISSUER"] = "ByIssuer";
})(EnumPaymentInterest = exports.EnumPaymentInterest || (exports.EnumPaymentInterest = {}));
var EnumCardType;
(function (EnumCardType) {
    EnumCardType["CREDIT"] = "CreditCard";
    EnumCardType["DEBIT"] = "DebitCard";
})(EnumCardType = exports.EnumCardType || (exports.EnumCardType = {}));
var EnumBrands;
(function (EnumBrands) {
    EnumBrands["VISA"] = "VISA";
    EnumBrands["MASTER"] = "MASTER";
    EnumBrands["AMEX"] = "AMEX";
    EnumBrands["ELO"] = "ELO";
    EnumBrands["AURA"] = "AURA";
    EnumBrands["JCB"] = "JCB";
    EnumBrands["DINERS"] = "DINERS";
    EnumBrands["DISCOVERY"] = "DISCOVER";
    EnumBrands["DISCOVER"] = "DISCOVER";
    EnumBrands["HIPERCARD"] = "HIPERCARD";
})(EnumBrands = exports.EnumBrands || (exports.EnumBrands = {}));
var EnumRecurrentPaymentInterval;
(function (EnumRecurrentPaymentInterval) {
    EnumRecurrentPaymentInterval["MONTHLY"] = "MONTHLY";
    EnumRecurrentPaymentInterval["BIMONTHLY"] = "BIMONTHLY";
    EnumRecurrentPaymentInterval["QUARTERLY"] = "QUARTERLY";
    EnumRecurrentPaymentInterval["SEMIANNUAL"] = "SEMIANNUAL";
    EnumRecurrentPaymentInterval["ANNUAL"] = "ANNUAL";
})(EnumRecurrentPaymentInterval = exports.EnumRecurrentPaymentInterval || (exports.EnumRecurrentPaymentInterval = {}));
var EnumRecurrentPaymentUpdateInterval;
(function (EnumRecurrentPaymentUpdateInterval) {
    EnumRecurrentPaymentUpdateInterval[EnumRecurrentPaymentUpdateInterval["MONTHLY"] = 1] = "MONTHLY";
    EnumRecurrentPaymentUpdateInterval[EnumRecurrentPaymentUpdateInterval["BIMONTHLY"] = 2] = "BIMONTHLY";
    EnumRecurrentPaymentUpdateInterval[EnumRecurrentPaymentUpdateInterval["QUARTERLY"] = 3] = "QUARTERLY";
    EnumRecurrentPaymentUpdateInterval[EnumRecurrentPaymentUpdateInterval["SEMIANNUAL"] = 6] = "SEMIANNUAL";
    EnumRecurrentPaymentUpdateInterval[EnumRecurrentPaymentUpdateInterval["ANNUAL"] = 12] = "ANNUAL";
})(EnumRecurrentPaymentUpdateInterval = exports.EnumRecurrentPaymentUpdateInterval || (exports.EnumRecurrentPaymentUpdateInterval = {}));
